Assets/HoloToolkit/Utilities/Scripts/Editor/ProjectSettingsWindow.cs

```diff
@@ -150,72 +224,48 @@ private void UpdateSettings(BuildTarget currentBuildTarget)
                 PlayerSettings.WSA.SetCapability(PlayerSettings.WSACapability.PrivateNetworkClientServer, false);
             }
 
-            var inputManagerPath = Directory.GetParent(Path.GetFullPath(Application.dataPath)).FullName + "\\ProjectSettings\\InputManager.asset";
-            bool userPermission = Values[ProjectSetting.XboxControllerSupport];
+            bool useToolkitAxes = Values[ProjectSetting.UseInputManagerAxes];
 
-            if (userPermission)
+            if (useToolkitAxes != EditorPrefsUtility.GetEditorPref(Names[ProjectSetting.UseInputManagerAxes], false))
             {
-                userPermission = EditorUtility.DisplayDialog("Attention!",
```

Comment:
I forget, but did we ever add the option to get a clean InputManager.asset or does this only add to the current mappings?  What happens if users do this more than once? Do we have a way to prevent duplicate entries?

Reply:
This only adds to the current mappings. Is there a downside to leaving the default mappings?

Reply:
I prefer to have a way to get a clean copy without all the extra mappings I won't need.

Reply:
I'm a bit on the fence.  Removing existing mappings could lead to issue, especially since most devs are already familiar and use them.
Better we simply manage our own and not interfere with other settings imho.

Reply:
I like having the option.